langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -248,6 +345,19 @@ public isolated function fromJsonDecimalString(string str) returns JsonDecimal|e
 # This works the same as function `cloneWithType`,
 # except that it also does the inverse of the conversions done by `toJson`.
 #
+# ```ballerina
+# json jsonNumberArray = [1, 2, 3, 4];
+# int[] intNumberArray = check jsonNumberArray.fromJsonWithType();
+# intNumberArray ⇒ [1,2,3,4]
+#
+# type Vowels string:Char[];
+#
+# json vowels = ["a", "e", "i", "o", "u"];
+# vowels.fromJsonWithType(Vowels) ⇒ ["a", "e", "i", "o", "u"];
```

Comment:
isn't the result of this expression is `["a","e","i","o","u"]`?

Reply:
yes, I will change.